### PR TITLE
Feat/moving ipfs to backend

### PR DESCRIPTION
**Pull request by reisr3**
- Moved IPFS operations to the backend
- Moved validation to the backend
- Added new job for `.env.dev` file so we can easily switch back and forth between environments
- Small bug fixes

Stacked behind this one:
https://github.com/engineer-dao/platform/pull/139

### CODE REVIEW

File: backend/interfaces/IJobData.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { JobState } from 'enums/JobState';
```
Comment (reisr3) [inline backend/interfaces/IJobData.ts:1]: In a coming ticket, we should move all the types into a root-level `shared` directory.

Comment (deweller) [reply backend/interfaces/IJobData.ts:1]: Yes.  I'm starting to run into that too...

**Comment by deweller** [comment]
Changing base to [feat/inline-status-and-loading](https://github.com/engineer-dao/platform/tree/feat/inline-status-and-loading) in order to review.

**Comment by deweller** [comment]
Nevermind - that didn't work.

File: backend/pages/api/ipfs/add.ts [hunk 2]
```diff
@@ -12,46 +15,32 @@ const cors = Cors({
...
+  if (!raw) {
     return res.status(422).json({ message: 'No message' });
   }
   // validate maximum size to prevent abuse
-  const metadataJSONString = JSON.stringify(metadata);
+  const metadataJSONString = JSON.stringify(raw);
+
   if (metadataJSONString.length > MAX_MESSAGE_SIZE) {
     return res.status(422).json({ message: 'Message too long' });
   }
```
Comment (deweller) [inline backend/pages/api/ipfs/add.ts:42]: We can get rid of this now that the JSON validation is in place.  This was just a stop-gap.

**Review by deweller — APPROVED** [review]
This looks good to me.

This makes me want to share form types and enums...

**Pull request merged**